generator/internal/language/templates/rust/crate/src/transport.rs.mustache

```diff
@@ -56,10 +56,21 @@ impl crate::traits::{{NameToPascal}} for {{NameToPascal}} {
     ) -> Result<{{OutputTypeName}}> {
         let options = options.set_default_idempotency(reqwest::Method::{{HTTPMethod}}.is_idempotent());
         let builder = self.inner.builder(
-            reqwest::Method::{{HTTPMethod}}, format!("{{HTTPPathFmt}}"
-               {{#HTTPPathArgs}}
-               , {{{.}}}
-               {{/HTTPPathArgs}}
+            reqwest::Method::{{HTTPMethod}},
+            format!("{{#HTTPPathInfo.PathTemplate}}{{#Literal}}/{{Value}}{{/Literal}}{{#FieldPath}}/{}{{/FieldPath}}{{#Verb}}:{{Value}}{{/Verb}}{{/HTTPPathInfo.PathTemplate}}"
+            {{#HTTPPathInfo.PathTemplate}}{{#FieldPath}},
+                req
+                {{#Components}}
+                    {{#Reference}}
+                        {{#Optional}}
+                            .{{Codec.FieldName}}.as_ref().ok_or_else(|| gax::error::Error::other("field \"{{Codec.FieldName}}\" is required in a request, but was missing"))?
+                        {{/Optional}}
+                        {{^Optional}}
+                            .{{Codec.FieldName}}
+                        {{/Optional}}
+                    {{/Reference}}
+                {{/Components}}
```

Comment:
Mustache preserves the indentation of your sections, so don't use indentation unless you want this to be in the generated code.  This may be closer to what we want:

```suggestion
                req
                {{#Components}}
                {{#Reference}}
                    {{#Optional}}
                    .{{Codec.FieldName}}.as_ref().ok_or_else(|| gax::error::Error::other("field \"{{Codec.FieldName}}\" is required in a request, but was missing"))?
                    {{/Optional}}
                    {{^Optional}}
                    .{{Codec.FieldName}}
                    {{/Optional}}
                {{/Reference}}
                {{/Components}}
```